bool xfm_reset
bool xfm_wakeup_disable
#int32 xfm_listen_angle
#int32 xfm_call_mode
